ui/oopsyraidsy/damage_tracker.ts

```diff
@@ -141,15 +163,40 @@ export class DamageTracker {
     const splitLine = e.line;
     const type = splitLine[0];
 
+    // If we're waiting on a timestamp callback, check if any have passed with this line.
+    // Ignore game log lines, which don't track milliseconds.
+    if (type !== '00') {
+      let timestampCallback = this.timestampCallbacks[0];
+      while (timestampCallback) {
+        const timeField = splitLine[1];
+        if (!timeField)
+          break;
+        const thisTimestamp = new Date(timeField).getTime();
+        if (thisTimestamp < timestampCallback.timestamp)
+          break;
```

Comment:
Is there any guarantee that callbacks are sorted by timestamp?

Reply:
Ah!  I appreciate you catching that.